src/components/BallotCard.jsx

```diff
@@ -399,6 +414,9 @@ export class BallotCard extends React.Component {
         let { contractsStore, votingType, children, isSearchPattern } = this.props;
         let isFromSearch = (this.isCreatorPattern() || this.isMemoPattern() || isSearchPattern);
         let ballotClass = (this.showCard() && isFromSearch) ? this.isFinalized ? "ballots-i" : "ballots-i ballots-i-not-finalized" : "ballots-i display-none";
+        let voteScaleClass = this.isFinalized ? "vote-scale" : "vote-scale vote-scale-not-finalized";
+        let hasAlreadyVotedLabel = <div type="button" className="ballots-i--vote ballots-i--vote-label ballots-i--vote-label-right ballots-i--vote_no">You already voted</div>;
```

Comment:
Should this `div` be without `type="button"`?

Reply:
yes, I'll remove